static/js/invite.js

```diff
@@ -163,6 +162,14 @@ exports.launch = function () {
             hashchange.exit_overlay();
         },
     });
+
+    autosize($("#invitee_emails").trigger("focus"));
+
+    $(document).on("keydown", (e) => {
+        if (e.which === 13 && e.ctrlKey) {
```

Comment:
We should somehow make clear that this is about the enter key. So e.g. add a comment `// enter key + ctrl` or perhaps better use `e.key === 'Enter'` instead of `e.which === 13`.

Reply:
I put the comment and used e.key instead of e.which.
Are any other corrections/edits required?
Thank you!